main.py

```diff
@@ -75,7 +76,7 @@ def _get_page_content(self, pathname: str) -> html.Div:
         try:
             if pathname == "/":
                 return self._make_matrix_layout()
-            elif pathname.startswith("/chart"):
+            elif pathname == "/chart" or pathname.startswith("/chart?"):
```

Comment:
I still believe that splitting the `pathname` on `?` and then the first component on `/` is a safer and more correct solution.

Reply:
Not sure here, the test here is simple and covers all cases we need, so why splitting the string....?